ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -136,6 +148,99 @@ class MediaPlayerService : Service() {
         }
     }
 
+    override fun onGetRoot(
+        clientPackageName: String,
+        clientUid: Int,
+        rootHints: Bundle?
+    ): MediaBrowserServiceCompat.BrowserRoot {
+
+        // Returns a root ID that clients can use with onLoadChildren() to retrieve
+        // the content hierarchy. Note that this root isn't actually displayed.
+        return MediaBrowserServiceCompat.BrowserRoot(MEDIA_BROWSER_ROOT_ID, null)
+    }
+
+    override fun onLoadChildren(
+        parentMediaId: String,
+        result: MediaBrowserServiceCompat.Result<List<MediaBrowserCompat.MediaItem>>
+    ) {
+
+        val mediaItems: MutableList<MediaBrowserCompat.MediaItem> = mutableListOf()
+
+        if (MEDIA_BROWSER_ROOT_ID == parentMediaId) {
```

Comment:
This part would be like the Main Menu. As you can see I only made one entry called "Browse Albums." According to the Android Auto docs there should only be like 4 entries in the root, so we'd have to choose wisely. There's also newer UI that has some tabs, which we should opt into and start to use. But what should be on these tabs and main menu? This part feels like there needs to be some community agreement.